index.js

```diff
@@ -11,7 +15,9 @@ import {Ed25519VerificationKey2020} from
 import {cryptosuite as eddsaRdfc2022CryptoSuite} from
   '@digitalbazaar/eddsa-rdfc-2022-cryptosuite';
 import examples2Context from './contexts/credentials/examples/v2';
-import {extendContextLoader} from 'jsonld-signatures';
+
+const TAB_TYPES = ['ecdsa-sd-2023', 'Ed25519Signature2020', 'eddsa-rdfc-2022',
+  'vc-jwt'] ;
```

Comment:
I don't think we need `Ed25519Signature2020` in here as part of the defaults, because it is the legacy format for `eddsa-rdfc-2022`. It can be used in specific examples, but not needed as a default. Ideally `ecdsa-rdfc-2019` would be added as part of the default here.

```suggestion
const TAB_TYPES = ['ecdsa-sd-2023', 'eddsa-rdfc-2022','vc-jwt'];
```

Reply:
Solved in dab6ebb which includes instructions for use.